app/src/main/java/social/entourage/android/sidemenu/SideMenuFragment.java

```diff
@@ -114,6 +118,12 @@ public void userInfoUpdated(Events.OnUserInfoUpdatedEvent event) {
     // ----------------------------------
 
     private void initialiseView() {
+        if(BuildConfig.FLAVOR_env=="staging") {
```

Comment:
Would it not be better to always set the text, but make it visible if you are in debug / invisible in release ?
Because checking for flavors through code isn't a very good practice

Reply:
It is only visible in some flavor (staging) not related to Debug

Reply:
Then it should have a layout specifically for this flavor. And same thing for the class. 
One layout/source set for the main resources, and one for the staging